src/expandableCalendar/agendaList.js

```diff
@@ -33,7 +33,9 @@ class AgendaList extends Component {
     /** whether to mark today's title with the "Today, ..." string. Default = true */
     markToday: PropTypes.bool,
     /** style passed to the section view */
-    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
+    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
+    /** whether to block the date change in calendar (and calendar context provider) when agenda scrolls */
+    blockDateUpdates: PropTypes.bool
```

Comment:
ok, so know that I understand the purpose of this prop. 
The new prop name is a little confusing, what about avoidDateUpdates? WDYT?

Reply:
Yes, I wasn't sure about the naming either. Sounds good.